ubg/src/index.js

```diff
@@ -5,9 +5,7 @@ import App from './App';
 import * as serviceWorker from './serviceWorker';
 
 ReactDOM.render(
-    <React.StrictMode>
```

Comment:
Is there a reason why React.StrictMode was deleted?

Reply:
Yep, it kept giving me warnings and it seems that StrictMode is going to be removed from React in the future.